### PR TITLE
Add set_logged and vacuum

### LINKED ISSUE #51: Write vacuum/analyze function
This could look like `def vacuum(conn_str, analyze=False, tables=None)`, where `tables` is a collection of tables to analyze, and, if not `tables`, then all tables in the pedsnet data model are vacuumed.  Given how we use it, it kind of makes sense to have `analyze` be the default, but that conflicts with the mental model "people" have of the Postgres command.

### LINKED ISSUE #53: Write set tables `LOGGED` function
This will be needed for after the transformation and also after the merge.

### PR DESCRIPTION
**Pull request by aaron0browne**
Fixes #51
Fixes #53

Unfortunately, this code needs more refinement. It should probably not
be in utils and also could use much better error handling. The tests are
extremely simple but they are better than nothing. Useable.

Signed-off-by: Aaron Browne <aaron0browne@gmail.com>

### CODE REVIEW

File: pedsnetdcc/tests/utils_test.py [hunk 1]
```diff
@@ -178,3 +199,64 @@ def test_user_at_end(self):
...
+        self.metadata.tables['person'].create(self.engine)
+
+        stmt = Statement('ALTER TABLE person SET UNLOGGED')
+        stmt.execute(self.conn_str)
+        self.assertFalse(stmt.err)
+
+        set_logged(self.conn_str, self.model_version, tables=['person'])
+
+        stmts = StatementList()
+        stmts.append(Statement('SELECT pg_is_in_recovery'))
```
Comment (murphyke) [inline pedsnetdcc/tests/utils_test.py]: Interesting, how does this test work?  Is there a link describing the technique?  I had also found this re: detected logged/unlogged: http://stackoverflow.com/questions/29899359/how-to-check-table-unlogged-with-postgresql

File: pedsnetdcc/utils.py [hunk 2]
```diff
@@ -286,3 +286,102 @@ def stock_metadata(model_version):
...
+    for stmt in stmts:
+        if stmt.err:
+            raise DatabaseError(
+                'setting tables to logged: {}: {}'.format(stmt.sql, stmt.err))
+
+
+# TODO: I'm not sure this belongs in utils since it executes SQL.
+def vacuum(conn_str, model_version, analyze=False, vocabulary=False,
+           tables=None):
+    """VACUUM (and optionally ANAYLZE) tables in a PEDSnet database
```
Comment (murphyke) [inline pedsnetdcc/utils.py]: typo

File: pedsnetdcc/utils.py [hunk 3]
```diff
@@ -286,3 +286,102 @@ def stock_metadata(model_version):
...
+    :param bool analyze:      whether to ANALYZE or not
+    :param bool vocabulary:   whether to operate on vocabulary tables
+    :param list(str) tables:  list of table names to operate on (overrides)
+    :return:
+    :raises DatabaseError:    if any of the SQL statements cause an error
+    """
+
+    from pedsnetdcc.db import Statement, StatementSet
+
+    table_names = tables or []
```
Comment (murphyke) [inline pedsnetdcc/utils.py]: Is this line necessary?

**Comment by murphyke** [comment]
Done reviewing

File: pedsnetdcc/tests/utils_test.py (hunk 1 above)
Comment (aaron0browne) [inline pedsnetdcc/tests/utils_test.py]: I got it from [here](http://michael.otacoo.com/postgresql-2/postgres-9-5-feature-highlight-alter-table-unlogged/) but it definitely doesn't work. I thought the `pg_is_in_recovery` **put** the db into recovery, but it obviously doesn't. I will fix with your suggestion.

**Comment by aaron0browne** [comment]
Ready for re-review.

**Comment by murphyke** [comment]
Looks good, merging.

**Pull request merged**